src/python/pants/backend/python/dependency_inference/BUILD

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Bikeshed: should this folder be `backend/python/dep_inference` or `backend/python/dependency_inference`?

Reply:
I have no opinion, so might as well leave it as-is.

Reply:
I vote for `backend/python/dependency_inference`

Reply:
Or consider calling it `extract_imports` even (unless you're planning on putting the `@rules` that will choose dependencies based on imports in this module as well?)

Reply:
> unless you're planning on putting the @rules that will choose dependencies based on imports in this module as well?

Yes, the plan is to add something like `dependency_inference/rules.py`. I'm not certain which files we'll have yet, but I want the logic to live together and separate from `backend/python/rules` because that has gotten too crowded.